smart-contracts/contracts/PublicLock.sol

```diff
@@ -60,6 +62,7 @@ contract PublicLock is
     MixinLockMetadata._initializeMixinLockMetadata(_lockName);
     MixinERC721Enumerable._initializeMixinERC721Enumerable();
     MixinRefunds._initializeMixinRefunds();
+    MixinLockManagerRole._initializeMixinLockManagerRole(_owner);
```

Comment:
maybe give this role to the beneficiary?

Reply:
at this point beneficiary == owner